ext/xmlwriter/tests/xmlwriter_toStream_encoding_gbk.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+XMLWriter::toStream() with encoding - test GBK
+--EXTENSIONS--
+xmlwriter
+--FILE--
+<?php
+
+$h = fopen("php://output", "w");
+
+$writer = XMLWriter::toStream($h);
+$writer->startDocument(encoding: "GBK");
+$writer->writeComment('ééé');
```

Comment:
Is it possible to use codepoint escape sequences? As my IDE loves messing up mixed encoding files